tests/samplers_tests/test_cmaes.py

```diff
@@ -61,6 +61,33 @@ def test_init_cmaes_opts(use_separable_cma: bool, cma_class_str: str) -> None:
         assert actual_kwargs["population_size"] is None
 
 
+@patch("optuna.samplers._cmaes.get_warm_start_mgd")
+def test_warm_starting_cmaes(mock_func_ws: MagicMock) -> None:
+    def objective(trial: optuna.Trial) -> float:
+        x = trial.suggest_uniform("x", -10, 10)
+        y = trial.suggest_uniform("y", -10, 10)
```

Comment:
Could you use `suggest_float` instead of `suggest_uniform` (c.f., #1470)?
```suggestion
        x = trial.suggest_float("x", -10, 10)
        y = trial.suggest_float("y", -10, 10)
```